apisupport.ant/src/org/netbeans/modules/apisupport/project/ui/ModuleActions.java

```diff
@@ -121,14 +123,9 @@ public ModuleActions(NbModuleProject project) {
 
     public void refresh() {
         Set<String> supportedActionsSet = new HashSet<String>();
-        globalCommands.put(ActionProvider.COMMAND_BUILD, new String[] {"netbeans"}); // NOI18N
+        globalCommands.put(ActionProvider.COMMAND_BUILD, new String[] {"build"}); // NOI18N
         globalCommands.put(ActionProvider.COMMAND_CLEAN, new String[] {"clean"}); // NOI18N
-        //a gross hack to prevent 158481 and the like
```

Comment:
I think @mkleint has to review this one!

Reply:
lol, was it me annoyed by the continually broken test classes?